Assets/BossRoom/Scripts/Client/UI/QualityButton.cs

```diff
@@ -0,0 +1,36 @@
+using TMPro;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Client
+{
+    public class QualityButton : MonoBehaviour
+    {
+        public TMP_Text qualityBtnText;
```

Comment:
Should be renamed to QualityBtnText

Reply:
You're right, I got confused with properties.
However, could it be made private since it is not used outside this class?

Reply:
Absolutely, good catch. Should be made private.